front-end/src/views/function/components/AddFunc.vue

```diff
@@ -0,0 +1,242 @@
+<template>
+  <el-drawer
+    ref="addFuncDrawer"
+    size="40%"
+    title="AddFunction"
+  >
+    <el-form ref="addForm"
+             class="addFunc"
+             :rules="rules"
+             :model="form"
+             label-width="80px"
+             label-position="top"
+             style="width: 90%;margin: 25px"
+             size="small"
+             :inline="true" >
+      <el-form-item label="FunctionName:"
+                    style="width: 100%"
+                    prop="functionName">
+        <el-input v-model="form.functionName"
+                  placeholder="请输入FunctionName"
+                  clearable>
+        </el-input>
+      </el-form-item>
+      <a style="color: red;font-size: 14px;">
+        *&nbsp;
+      </a>
+      <a style="font-size: 14px;">
+        Inputs:
+      </a>
+      <el-form-item style="width: 100%;margin-top: 5px"
+                    v-for="(value,index) in form.inputs"
+                    :prop="'inputs.'+index"
+                    :key="index"
```

Comment:
Codacy has a fix for the issue: [Replace `····················` with `········`](https://app.codacy.com/gh/FunctionStream/function-stream/pullRequest?prid=7834759)
```suggestion
        :key="index"
```